### ISSUE TITLE
Invalid Auth Tokens cause some funkiness

### ISSUE THREAD

**Issue by jasonbahl** [wp-graphql/wp-graphql-jwt-authentication#21]
When invalid Auth tokens are passed in the headers funkiness ensues. 

When using GraphiQL, if I have expired headers in my Request, even the Schema generation takes quite some time. 

I think it's trying to constantly re-validate the headers or something. Not entirely sure, but _something_ is funky and needs to be looked at a bit more.

**Comment by mdbitz** [comment]
Hi @jasonbahl :

I've duplicated this behavior on a site using the plugin. Within the `validate_token` function if an invalid Auth Token is found it triggers an error using the `__()` function. `__()` in turn call for translation and makes a `wp_get_current_user` call and causes an infinite loop within the `validate_token` function.

Does it make sense to keep the translated Error Messages? If so it would be good to remove the determine_current_user filter before calling `validate_token` then adding it back in after.

https://github.com/wp-graphql/wp-graphql-jwt-authentication/blob/master/src/Auth.php#L376

```
remove_filter(
    'determine_current_user', 
    [
        '\WPGraphQL\JWT_Authentication\Auth',
        'filter_determine_current_user'
   ], 
   10
);
```

**Comment by jasonbahl** [comment]
This issue is resolved by the work done to address #38, which will be released shortly.
